vector/src/main/scala/geotrellis/vector/package.scala

```diff
@@ -23,7 +23,19 @@ import org.locationtech.jts.{geom => jts}
 package object vector extends SeqMethods
     with reproject.Implicits
     with triangulation.Implicits
-    with voronoi.Implicits {
+    with voronoi.Implicits
+    with io.json.Implicits
+    with io.wkt.Implicits
+    with io.wkb.Implicits {
+
+  /** The algorithms herein are all implemented in JTS, but the wrapper methods
+    * here make it straightforward to call them with geotrellis.vector classes.
+    */
+  implicit class withAnyGeometryMethods[G <: Geometry](val self: G) extends MethodExtensions[G]
```

Comment:
I moved this up just so it's easier to see what implicits will be in scope when importing `geotrellis.vector._`